syntax = "proto2";

message RegistrationSuccessNotificationMessage {
    required string serialNumber = 2;
}